### ISSUE TITLE
which architecture should be selected for finetuning transformer_lm.wmt19.en

### ISSUE THREAD

**Issue by Sohyo** [pytorch/fairseq#2368]
Hello

I am trying to finetune transformer_lm.wmt19.en on different domain data. 

`fairseq-train --task language_modeling \
  data-bin/wmt_data \
  --restore-file data-bin/wmt19.en/model.pt \
  --save-dir checkpoints/finetune_wmt19 \
  --arch transformer_lm --share-decoder-input-output-embed \
  --dropout 0.1  \
  --optimizer adam --adam-betas '(0.9, 0.98)'  --weight-decay 0.01 --clip-norm 0.0\
  --lr 0.0005 --lr-scheduler inverse_sqrt --warmup-updates 4000 --warmup-init-lr 1e-07 \
  --tokens-per-sample 512 --sample-break-mode none \
  --max-sentences 16  --update-freq 16 \
  --fp16 \
  --max-update 500`

However, seems like **transformer_lm** is not the proper architecture.

> "please ensure that the architectures match.".format(filename)
Exception: Cannot load model parameters from checkpoint data-bin/wmt19.en/model.pt; please ensure that the architectures match.

I also tried with other architectures but I couldn't find the proper architecture.
Could you help me with this?
